src/NServiceBus.AcceptanceTests/NonDTC/When_sending_a_durable_message.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace NServiceBus.AcceptanceTests.NonDTC
+{
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Configuration.AdvanceExtensibility;
+    using NServiceBus.Features;
+    using NServiceBus.ObjectBuilder;
+    using NServiceBus.Transports;
+    using NServiceBus.Unicast;
+    using NUnit.Framework;
+
+    public class When_sending_a_durable_message : NServiceBusAcceptanceTest
```

Comment:
is this a test we already have in v6+ or should we add this test to develop as well?

Reply:
Yes, we have a unit test for this in v6+ https://github.com/Particular/NServiceBus/blob/support-6.2/src/NServiceBus.Core.Tests/Reliability/Outbox/TransportReceiveToPhysicalMessageProcessingConnectorTests.cs#L45